GVFS/GVFS.Common/Maintenance/GitMaintenanceScheduler.cs

```diff
@@ -54,6 +54,13 @@ private void ScheduleRecurringSteps()
                     state: null,
                     dueTime: prefetchPeriod,
                     period: prefetchPeriod));
+
+                TimeSpan looseObjectsPeriod = TimeSpan.FromHours(12);
+                this.stepTimers.Add(new Timer(
+                    (state) => this.queue.TryEnqueue(new LooseObjectsStep(this.context, requireCacheLock: true)),
+                    state: null,
+                    dueTime: looseObjectsPeriod,

```

Comment:
Should the `dueTime` be sooner than 12 hours?  

If someone has their machine configured to take Windows updates at 6 or 7pm every night, GVFS.Mount.exe might never hit the `dueTime` (assuming they long in after 7am in the morning).

Reply:
Yes, these steps with a long period should have a short dueTime.

Reply:
@derrickstolee 6 hours?

Reply:
I was thinking on the order of 15 seconds.

Reply:
That would be a lot of churn in the background.  Opening the files reading the date, etc...  we could go to 1 hour if we want to be agressive

Reply:
The `dueTime` is just for the first trigger. The `period` is used for the frequency.

Reply:
For these bigger maintenance steps, we should have _different_ values for `dueTime` and `period` (which is not necessary for background prefetch, as 15 minutes is good enough for both).

Reply:
We can play with the values.  I remember you said there is a race condition here, so maybe 5 minutes for due time on this one and we can do the check hourly.  That should be pretty cheap and def allow the maintence jobs to run.